Source/Helpers/YPDeviceOrientationHelper.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  YPDeviceOrientationHelper.swift
+//  YPImagePicker
+//
+//  Created by Kf on 2019/7/22.
+//  Copyright © 2019 Yummypets. All rights reserved.
+//
+
+import UIKit
+import CoreMotion
+
+// Reference: https://medium.com/@PabloDomine/developing-camille-how-to-determine-device-orientation-in-a-camera-app-4c622d251993
+class YPDeviceOrientationHelper {
+    static let shared = YPDeviceOrientationHelper() // Singleton is recommended because an app should create only a single instance of the CMMotionManager class.
```

Comment:
What if the user want to create it's own CMMotionManager?

Reply:
How about make CMMontionManager be the member variable of YPCameraVC to let the user can use his manager?

Reply:
@s4cha what do you think?

Reply:
From what I understand in the documentation this is better to have only one for performance reasons but having multiple multiple `CMMotionManager` is not forbidden per se.
It says "instances of this class can affect the rate at which data is received from the accelerometer and gyroscope." So I'd say it's not a big deal for now, if an issue pops then we'll deal with it :)